kinto/tests/test_views_collections.py

```diff
@@ -35,12 +35,15 @@ def test_collections_name_should_be_simple(self):
                           headers=self.headers,
                           status=400)
 
-    def test_unknown_bucket_raises_404(self):
+    def test_unknown_bucket_raises_403(self):
```

Comment:
> t also have a disadvantage: It leaks the fact that the resource exists to another user, which could then use this information in many different ways.

As I said in this case the resource doesn't exists so it obviously doesn't disclose the fact that it exists :)